source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -272,13 +280,6 @@ def _get_TextInfo(self):
 		bounds review to the visible text."""
 		return consoleUIATextInfo
 
-	def _getTextLines(self):
```

Comment:
I assume this now works when inherited from the base class, and that's the reason why you removed this?

Reply:
This is correct. The base implementation of _getTextLines seems to be fine as far as I understand.